src/caret_analyze/plot/visualize_lib/bokeh/timeseries.py

```diff
@@ -66,20 +67,51 @@ def create_figure(self) -> Figure:
         # Apply xaxis offset
         records_range = Range([to.to_records() for to in target_objects])
         frame_min, frame_max = records_range.get_range()
```

Comment:
```suggestion
        x_range_name = None
        frame_min, frame_max = records_range.get_range()
```

Reply:
Please refer to the following for details.
https://github.com/tier4/CARET_analyze/pull/276#discussion_r1204935711

Reply:
From the difference between the return values of `records_range.get_range()` and `converter.convert()`, it is better to have another variable for `flame_min/max` and `flame_min/max_contert`.